sql/planbuilder/aggregates.go

```diff
@@ -348,14 +352,14 @@ func (b *Builder) buildAggregateFunc(inScope *scope, name string, e *ast.FuncExp
 		aggType = types.Float64
 	}
 
-	aggName := strings.ToLower(agg.String())
+	aggName := strings.ToLower(plan.AliasSubqueryString(agg))
```

Comment:
Very skeptical this new method is being used in place of expression.String() everywhere it could matter

You have a plan out of this mess?

Reply:
expressions are given an expression id, the expression id is used for setting execution indexes